lib/src/lib.rs

```diff
@@ -635,23 +641,28 @@ impl Repo {
             path: repo_path.to_owned(),
             pub_key: pk,
             version: 0,
+            chunking: config::RepoChunking::default(),
         };
         Ok(repo)
     }
 
     /// Create new rdedup repository
-    pub fn init(repo_path: &Path, passphrase: &str) -> Result<Repo> {
+    pub fn init(repo_path: &Path,
+                passphrase: &str,
+                chunking: config::RepoChunking)
+                -> Result<Repo> {
         info!("Init repo {}", repo_path.to_string_lossy());
         let (pk, sk) = box_::gen_keypair();
 
         Repo::ensure_repo_not_exists(repo_path)?;
         Repo::init_common_dirs(repo_path)?;
-        config::write_config_v1(repo_path, &pk, &sk, passphrase)?;
+        config::write_config_v1(repo_path, &pk, &sk, passphrase, chunking.clone())?;
```

Comment:
This `.clone` seems out of place. I guess all types in `config::` could just have `#[derive(Copy)]` instead.